apps/src/templates/teacherDashboard/RosterDialog.jsx

```diff
@@ -81,15 +81,21 @@ const ClassroomList = ({classrooms, onSelect, selectedId}) => classrooms.length
     <p>
       {locale.noClassroomsFound()}
     </p>
-    <a href="https://classroom.google.com/">
-      {locale.addRemoveGoogleClassrooms()}
-    </a>
+    {provider === 'google' ?
+      <a href="https://classroom.google.com/">
+        {locale.addRemoveGoogleClassrooms()}
+      </a> :
+      <a href="https://clever.com/">
+        {locale.addRemoveCleverClassrooms()}
+      </a>
+    }
```

Comment:
I wonder if this component has grown in complexity enough to deserve it's own module. I also personally have a general antipathy towards using lots of ternaries in JSX (there are a couple levels of it here).